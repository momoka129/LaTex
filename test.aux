\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Formulas}{2}{section.1}\protected@file@percent }
\newlabel{eq0}{{1}{2}{Formulas}{equation.1.1}{}}
\gdef \@abspage@last{2}
