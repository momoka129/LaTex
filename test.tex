\documentclass[12pt]{article}
\usepackage{hyperref}  % add this package to support \autoref
\usepackage{amsmath}
\begin{document}
Hello world! This is my first \LaTeX\ document. 
\newpage
\section{Formulas}
A rectangle has side lengths of $(x+1)$ and $(x+3)$.A hard return is going to start a new paragraph.\\
A rectangle has side lengths of $(x+1)$ and $(x+3)$. \textbackslash\textbackslash\ is a soft return and therefore the line is not indented.

The equation $${A(x)=x^2+4x+3}$$ gives the area of the rectangle.\\
\{\} makes sure to keep your equation on a line.

\begin{equation}\label{eq0}
    \alpha^2+\beta^2=\gamma^2
\end{equation}
Famous Gaussian quadrature:
\begin{equation}
    \begin{split}
        S&=1+2+3+\dots+n\\
        S&=n+(n-1)+(n-2)+\dots+1\\
        2S&=(1+n)+(2+(n-1))+(3+(n-2))+\dots+(n+1)\\
        2S&=n(n+1)\\
        S&=\frac{n(n+1)}{2} 
    \end{split}
\end{equation}
Formulas for various situations:
\begin{equation}
    F(x)=
    \begin{cases}
        0&,\text{if $x<-1$}\\
        x+1&,\text{if $x>3$}\\
        1&,\text{otherwise.}
    \end{cases}
\end{equation}

\[
    a^2+b^2=c^2
\]

Reference test \autoref{eq0}

Insertion of pictures:\\
Try to insert vector graphics so that the image will not change in clarity when it is enlarged or reduced.
\end{document} 