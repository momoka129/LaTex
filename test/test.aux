\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\bibstyle{apacite}
\citation{trevisanato2000tea}
\@writefile{toc}{\contentsline {section}{\numberline {1}Formulas}{3}{section.1}\protected@file@percent }
\newlabel{eq0}{{1}{3}{Formulas}{equation.1.1}{}}
\citation{mckay2002role}
\@writefile{toc}{\contentsline {section}{\numberline {2}Insertion of pictures}{4}{section.2}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces idk what}}{4}{figure.1}\protected@file@percent }
\newlabel{fig:a}{{1}{4}{idk what}{figure.1}{}}
\citation{yang1993tea}
\citation{yang1993tea}
\citation{yang1993tea}
\citation{yang1993tea}
\bibdata{test}
\bibcite{mckay2002role}{\citeauthoryear {McKay\ \BBA{} Blumberg}{McKay\ \BBA{} Blumberg}{{\APACyear {2002}}}}
\APACbibcite{mckay2002role}{\citeauthoryear {McKay\ \BBA{} Blumberg}{McKay\ \BBA{} Blumberg}{{\APACyear {2002}}}}
\bibcite{trevisanato2000tea}{\citeauthoryear {Trevisanato\ \BBA{} Kim}{Trevisanato\ \BBA{} Kim}{{\APACyear {2000}}}}
\APACbibcite{trevisanato2000tea}{\citeauthoryear {Trevisanato\ \BBA{} Kim}{Trevisanato\ \BBA{} Kim}{{\APACyear {2000}}}}
\bibcite{yang1993tea}{\citeauthoryear {Yang\ \BBA{} Wang}{Yang\ \BBA{} Wang}{{\APACyear {1993}}}}
\APACbibcite{yang1993tea}{\citeauthoryear {Yang\ \BBA{} Wang}{Yang\ \BBA{} Wang}{{\APACyear {1993}}}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Sheet}{5}{section.3}\protected@file@percent }
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces My first table}}{5}{table.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{References}{5}{section*.2}\protected@file@percent }
\gdef \@abspage@last{5}
