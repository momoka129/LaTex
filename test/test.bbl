\begin{thebibliography}{}

\bibitem [\protect \citeauthoryear {%
McKay%
\ \BBA {} Blumberg%
}{%
McKay%
\ \BBA {} Blumberg%
}{%
{\protect \APACyear {2002}}%
}]{%
mckay2002role}
\APACinsertmetastar {%
mckay2002role}%
\begin{APACrefauthors}%
McKay, D\BPBI L.%
\BCBT {}\ \BBA {} Blumberg, J\BPBI B.%
\end{APACrefauthors}%
\unskip\
\newblock
\APACrefYearMonthDay{2002}{}{}.
\newblock
{\BBOQ}\APACrefatitle {The role of tea in human health: an update} {The role of tea in human health: an update}.{\BBCQ}
\newblock
\APACjournalVolNumPages{Journal of the American College of Nutrition}{21}{1}{1--13}.
\PrintBackRefs{\CurrentBib}

\bibitem [\protect \citeauthoryear {%
Trevisanato%
\ \BBA {} Kim%
}{%
Trevisanato%
\ \BBA {} Kim%
}{%
{\protect \APACyear {2000}}%
}]{%
trevisanato2000tea}
\APACinsertmetastar {%
trevisanato2000tea}%
\begin{APACrefauthors}%
Trevisanato, S\BPBI I.%
\BCBT {}\ \BBA {} Kim, Y\BPBI I.%
\end{APACrefauthors}%
\unskip\
\newblock
\APACrefYearMonthDay{2000}{}{}.
\newblock
{\BBOQ}\APACrefatitle {Tea and health} {Tea and health}.{\BBCQ}
\newblock
\APACjournalVolNumPages{Nutrition reviews}{58}{1}{1--10}.
\PrintBackRefs{\CurrentBib}

\bibitem [\protect \citeauthoryear {%
Yang%
\ \BBA {} Wang%
}{%
Yang%
\ \BBA {} Wang%
}{%
{\protect \APACyear {1993}}%
}]{%
yang1993tea}
\APACinsertmetastar {%
yang1993tea}%
\begin{APACrefauthors}%
Yang, C\BPBI S.%
\BCBT {}\ \BBA {} Wang, Z\BHBI Y.%
\end{APACrefauthors}%
\unskip\
\newblock
\APACrefYearMonthDay{1993}{}{}.
\newblock
{\BBOQ}\APACrefatitle {Tea and cancer} {Tea and cancer}.{\BBCQ}
\newblock
\APACjournalVolNumPages{JNCI: Journal of the National Cancer Institute}{85}{13}{1038--1049}.
\PrintBackRefs{\CurrentBib}

\end{thebibliography}
