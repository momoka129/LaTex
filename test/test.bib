@article{trevisanato2000tea,
  title={Tea and health},
  author={Trevisanato, Siro I and Kim, Young In},
  journal={Nutrition reviews},
  volume={58},
  number={1},
  pages={1--10},
  year={2000},
  publisher={Oxford University Press Oxford, UK}
}
@article{mckay2002role,
  title={The role of tea in human health: an update},
  author={McKay, Diane L and Blumberg, Jeffrey B},
  journal={Journal of the American College of Nutrition},
  volume={21},
  number={1},
  pages={1--13},
  year={2002},
  publisher={Taylor \& Francis}
}
@article{yang1993tea,
  title={Tea and cancer},
  author={Yang, Chung S and Wang, Zhi-Yuan},
  journal={JNCI: Journal of the National Cancer Institute},
  volume={85},
  number={13},
  pages={1038--1049},
  year={1993},
  publisher={Oxford University Press}
}