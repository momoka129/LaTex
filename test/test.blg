This is BibTeX, Version 0.99d (TeX Live 2024)
Capacity: max_strings=200000, hash_size=200000, hash_prime=170003
The top-level auxiliary file: test.aux
The style file: apacite.bst
Reallocated glb_str_ptr (elt_size=4) to 20 items from 10.
Reallocated global_strs (elt_size=200001) to 20 items from 10.
Reallocated glb_str_end (elt_size=4) to 20 items from 10.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated wiz_functions (elt_size=4) to 6000 items from 3000.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated wiz_functions (elt_size=4) to 9000 items from 6000.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Database file #1: test.bib
apacite.bst [2013/07/21 v6.03 APA bibliography style]
You've used 3 entries,
            6951 wiz_defined-function locations,
            1291 strings with 11070 characters,
and the built_in function-call counts, 4614 in all, are:
= -- 598
> -- 42
< -- 137
+ -- 171
- -- 6
* -- 259
:= -- 723
add.period$ -- 12
call.type$ -- 3
change.case$ -- 51
chr.to.int$ -- 0
cite$ -- 6
duplicate$ -- 144
empty$ -- 217
format.name$ -- 72
if$ -- 956
int.to.chr$ -- 0
int.to.str$ -- 9
missing$ -- 0
newline$ -- 78
num.names$ -- 24
pop$ -- 74
preamble$ -- 1
purify$ -- 12
quote$ -- 0
skip$ -- 361
stack$ -- 0
substring$ -- 205
swap$ -- 90
text.length$ -- 6
text.prefix$ -- 0
top$ -- 1
type$ -- 240
warning$ -- 0
while$ -- 21
width$ -- 0
write$ -- 95
