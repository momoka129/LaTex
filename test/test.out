\BOOKMARK [1][-]{section.1}{\376\377\000F\000o\000r\000m\000u\000l\000a\000s}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000I\000n\000s\000e\000r\000t\000i\000o\000n\000\040\000o\000f\000\040\000p\000i\000c\000t\000u\000r\000e\000s}{}% 2
\BOOKMARK [1][-]{section.3}{\376\377\000S\000h\000e\000e\000t}{}% 3
\BOOKMARK [1][-]{section*.2}{\376\377\000R\000e\000f\000e\000r\000e\000n\000c\000e\000s}{}% 4
